### DISCUSSION TITLE
Is there a way to set the background color for the pane, per domain (E.g. local, Ubuntu, PowerShell)?

### DISCUSSION THREAD

**Discussion by Hammit** [wezterm/wezterm#5752] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I'm running on Windows. I WezTerm to run PowerShell and also my WSL distributions. I would like to be able to set the background color when in the powershell tab to blue, to indicate it is powershell (just like running powershell in Windows), otherwise black when I'm running Ubuntu in WSL.

Is this possible. I don't see how it is atm, but then again, that could be me :) Thanks for the help

**Comment by Parsifa1** [comment]
maybe you can do it like this:
```lua
wezterm.on("format-tab-title", function(tab, tabs, panes, config, hover, max_width)
	local color = { ["local"] = "#111fff" } -- change local to that domain for powershell
	local background = color[tab.active_pane.domain_name] 
	return {
		{ background = { color = background } },
		{ text = tab.active_pane.title },
	}
end)

```

**Reply by Hammit** [reply]
Thanks for your reply. It seems your answer only styles the tab. I'm looking to style the whole pane. Thanks tho. I honestly don't think it's possible to do what I'm wanting atm, so this is probably the only/best alternative

**Reply by wez** [reply]
wezterm doesn't currently support per-pane configuration overrides, so if you want to color a pane, you will need to emit escape sequences in that pane to change its palette.  https://wezfurlong.org/wezterm/escape-sequences.html#csi-control-sequence-introducer-sequences lists a number of options for this.  https://github.com/mbadolato/iTerm2-Color-Schemes/tree/master/dynamic-colors has a number of scripts that demonstrate this approach for a subset of the color schemes supported by wezterm.

**Reply by wez** [reply]
If you don't use split panes, then you can override the color for a window using https://wezfurlong.org/wezterm/config/lua/window/set_config_overrides.html from within the https://wezfurlong.org/wezterm/config/lua/window-events/update-status.html event
